### PR TITLE
article mobile bits

### PR DESCRIPTION
**Pull request by buddy-web3**

### CODE REVIEW

File: app/components/Article/article.css
Comment (jrhender) [inline app/components/Article/article.css:141]: A couple small typos
```suggestion
@media only screen and (max-width: 640px) {
  article .footer-container {
    flex-direction: row;
    flex-wrap: wrap;
  }
  article .footer-container > div:nth-child(-n + 2) {
    flex: 1 1;
  }
}
```

File: app/components/Article/article.css
Comment (jrhender) [inline app/components/Article/article.css:138]: Is it a good practice to use `div:nth-child` as used? Perhaps an additional class would be clearer when reading the code and less likely to break unexpectedly? I don't have a strong feeling here, just thought I would ask.
```suggestion
  article .footer-comtainer .footer-section {
```

My CSS knowledge is limited, so my apologies if the rationale for the current approach is obvious 😄

File: app/components/Article/article.css
Comment (buddy-web3) [reply app/components/Article/article.css:141]: @jrhender I noticed the typo as well.. but it's everywhere, not just in my additions.

File: app/components/Article/article.css
Comment (buddy-web3) [reply app/components/Article/article.css:138]: Both options are fine by me. @mruwnik do you prefer nth-child or a new class?

File: app/components/Article/article.css
Comment (jrhender) [reply app/components/Article/article.css:141]: Ah I see. Better to keep as is then and fix in another PR I suppose.

**Comment by melissasamworth** [comment]
All looks good to me, let's wait on a final decision re "div:nth-child" before merging

**Review by melissasamworth — APPROVED** [review]

**Pull request merged**
